examples/chat.py

```diff
@@ -65,6 +65,6 @@ def main(host, port):
         print('usage: python chat.py google.com 80')
         print('(if you use this example, try typing')
         print('GET /')
-        print('and then hitting enter)')
+        print('and then hitting enter twice.)')
```

Comment:
nice, the issue was that Google won't send you the HTTP response until you hit enter twice